pkg/sql/create_external_connection.go

```diff
@@ -144,6 +144,10 @@ func (p *planner) createExternalConnection(
 	// Create the External Connection and persist it in the
 	// `system.external_connections` table.
 	if err := ex.Create(params.ctx, txn); err != nil {
+		isNotExists := n.ConnectionLabelSpec.IfNotExists
```

Comment:
nit: s/is/if/r

also, I think a few words in the commit message could be swapped for clearer reading.